assets/src/settings-page/plugin-suppression.js

```diff
@@ -251,6 +255,11 @@ function PluginRow( { pluginKey, pluginDetails } ) {
 								dangerouslySetInnerHTML={ { __html: autop( pluginDetails.Description ) } }
 							/>
 						),
+						pluginDetails.PluginURI && (
+							<a href={ pluginDetails.PluginURI } target="_blank" rel="noreferrer">
```

Comment:
The `a` element requires a unique key:

![image](https://user-images.githubusercontent.com/16200219/89089174-901dac00-d38b-11ea-8ef7-48d92de19878.png)


```suggestion
							<a href={ pluginDetails.PluginURI } target="_blank" rel="noreferrer" key={ `${ pluginKey }-details-plugin-uri` }>
```